src/helpers/jit-helper.js

```diff
@@ -77,14 +77,18 @@ class JitHelper {
    */
   static _moduleSourceRefactoring(template) {
     const { module } = template;
-    const promises = Object.keys(module).filter(it => module[it]).map(it => {
-      const { source } = module[it];
-      if (source && source[0] === '.') {
-        module[it].source = path.normalize(path.resolve(template.locals.component.path, source));
-      }
-    });
+    if (module) {
+      const promises = Object.keys(module).filter(it => module[it]).map(it => {
+        const { source } = module[it];
+        if (source && source[0] === '.') {
```

Comment:
I guess it can be simplified into
```
        if (source) {
          module[it].source = path.resolve(template.locals.component.path, source);
        }
```
according to [docs](https://nodejs.org/docs/latest-v8.x/api/path.html#path_path_resolve_paths)